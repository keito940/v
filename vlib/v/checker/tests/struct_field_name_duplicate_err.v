struct A {
	a int
	a string
}
fn main(){}
