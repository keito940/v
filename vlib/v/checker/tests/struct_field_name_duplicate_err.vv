struct A {
	a int
	a string
}
