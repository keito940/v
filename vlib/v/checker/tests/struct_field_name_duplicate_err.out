vlib/v/checker/tests/struct_field_name_duplicate_err.v:3:4: error: field name `a` duplicate
    1| struct A {
    2|     a int
    3|     a string
             ~~~~~~
    4| }
    5| fn main(){}
